src/js/components/charts/Tooltip.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+import _ from 'lodash';
+
+import {
+    Hint
+} from 'react-vis';
+
+export default ({value, ...props}) => {
+    // TODO: This needs to be styled.
+    // `value` is a flat key-value object
+    return (
+        <Hint {...props} value={value}>
+          <div style={{background: 'black'}}>
+            <h4>Value of hint</h4>
+
+            {
+                _(value).map((v, k) => {
+                    // How each key-value is rendered
+                    return <p key={k}>{k}: {v.toString()}</p>;
+                }).value()
+            }
+
+          </div>
+        </Hint>
+    );
+};
+
+export const onValueChange = (datapoint, eventType, current, setCurrent) => {
+    console.log(`START: ${eventType}`);
+
+    if (!current ||
+        (current && (datapoint.x.toString() !== current.x.toString() ||
+                          datapoint.y !== current.y))) {
+        console.log('setting datapoint');
+        setCurrent(datapoint);
+    }
```

Comment:
`!current || current` is redundant, also, I'm not sure why is it needed the `toString()`; I'd purpose the more simple alternative:
```suggestion
  if (current && datapoint.x === current.x && datapoint.y === current.y) {
    return
  }
  
  console.log('setting datapoint');
  setCurrent(datapoint);
```

Reply:
the `toString` was necessary because wasn't working for dates. btw this has been fixed already in next PR